lib/unifex/specs.ex

```diff
@@ -70,26 +70,58 @@ defmodule Unifex.Specs do
     {_res, binds} = Code.eval_string(specs_code, [{:unifex_config__, []}], make_env(specs_file))
     config = binds |> Keyword.fetch!(:unifex_config__) |> Enum.reverse()
 
-    {functions_args, functions_results} =
+    {functions_args_arity, functions_results} =
       config
       |> Keyword.get_values(:function)
-      |> Enum.map(fn {name, args, results} -> {{name, args}, {name, results}} end)
+      |> Enum.map(fn {name, args, results} ->
+        {{{name, args}, {name, Enum.count(args)}}, {name, results}}
+      end)
       |> Enum.unzip()
 
+    {functions_args, functions_arity} = Enum.unzip(functions_args_arity)
```

Comment:
```suggestion
functions_arity  = Enum.map(functions_args, fn {name, args} -> {name, Enum.count(args)} end)
```
You don't have to create `functions_args_arity` variable, you can just create `function_args` the way it has been done before your changes and create `functions_arity` with a simple `Enum.map`, without second call of `Enum.unzip`, IMO it will simplify your code